### DISCUSSION TITLE
Unit 1.4 Part 4, Part 5

### DISCUSSION THREAD

**Discussion by may12day** [Lightning-AI/dl-fundamentals#4] [General discussions]
Couldn't able to watch video lecture for Unit 1.4 - Part 4 and Part 5 respectively? A blurry thumbnail is hovering around the video pane. 
Kindly fix it as soon as possible.

**Comment by rasbt** [comment]
Arg sorry, looking into this! 

Was trying it out myself and it seems fine. Could a browser refresh help perhaps? Nonetheless I will forward to the web devs!
<img width="1512" alt="Screenshot 2022-12-07 at 2 55 36 PM" src="https://user-images.githubusercontent.com/5618407/206293800-43f68ff2-888e-4a71-8590-c04eee66cbff.png">

**Comment by may12day** [comment]
Cleared the cache and cookies as well, seems browser specific problem. Worked for me in Firefox not in MS Edge. 

Here's the screenshot :
![1-4-The-First-Machine-Learning-Classifier-Lightning-AI](https://user-images.githubusercontent.com/26703868/206439723-d8ce0709-ba58-4845-8ac2-1844178977e8.png)
